entity-framework/core/get-started/xamarin.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Getting Started - EF Core
+author: masoucou
+ms.date: 07/07/2020
+uid: core/get-started/xamarin
+---
+
+# Getting Started with EF Core and Xamarin
+
+In this tutorial, you create a Xamarin.Forms application that performs data access against a SQLite database using Entity Framework Core.
```

Comment:
Can this first reference to Xamarin.Forms link to the documentation?

Reply:
Done.